libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -259,6 +259,13 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("BLEND_TC", 21, AP_GPS, _blend_tc, 10.0f),
 
+    // @Param: HDG_ENABLE
+    // @DisplayName: Use GPS yaw if available
+    // @Description: Enables the ArduPilot system to use the a true heading (yaw) from the GPS if available. This can come from some GPS types which can estimate heading when not moving. An example is some NMEA GPS units which support the HDT NMEA sentence.
```

Comment:
Would it be worth noting on this line that this requires EKF3 to take advantage of the information, or are the EKF param docs considered sufficient to convey that?

Reply:
I think we'll need external docs on how to use this. There are quite a few conditions for it.